### ISSUE TITLE
Checks, fixes and reviews

### ISSUE THREAD

**Issue by juankaromo** [wazuh/wazuh-kibana-app#1413]
Hi team,

We need to check, fix and change some points:

- [x] Check for error handler calls that don't have prefixes like "Routes."
- [x] In the checklist of a check on policy, if there are many or if one is too long it is cut off.
- [x] The SCA overview dashboard needs review, avoid giant metrics and think in other visualizations.
- [x] Show `follow_symbolic_link` in config on demand

Regards.

**Comment by juankaromo** [comment]
Working on this branch https://github.com/wazuh/wazuh-kibana-app/tree/check-fix-review

**Comment by juankaromo** [comment]
**Update 30/04/19**

For the incoming 3.9 release, we check error handler calls that don't have prefixes like "Routes.", we fix cut off checks in the checklist of a check on policy, and follow_symbolic_link is shown now in the config on demand options.

On the other hand, the SCA overview dashboard has been reviewed. We have removed giant visualizations, have made smaller other ones, and others have been added. Its final appearance is as follows: 
![image](https://user-images.githubusercontent.com/21195730/56969391-5b59ee00-6b65-11e9-8cf1-2fc2625921dd.png)
